next.clayui.com/content/docs/css/components/labels.md

```diff
@@ -0,0 +1,412 @@
+	<span class="label-item label-item-before">
+		<span class="sticker">
+			...
+		</span>
+	</span>
+	<span class="label-item label-item-before">
+		...
+	</span>
+	<span class="label-item label-item-before">
+		...
+	</span>
+	<span class="label-item label-item-expand">
+		<a href="#1">Label with Link</a>
+	</span>
+	<span class="label-item label-item-after">
+		<button aria-label="Close" class="close" type="button">
+			...
+		</button>
+	</span>
+</span>
+```
+
+#### Interactive Label
+
+Add the `tabindex="0"` attribute to the `.label` element to create an interactive label with multiple controls inside. The inner controls should have the attribute `tabindex="-1"` to remove them from the tab order. The inner controls can be placed back in the tab order by changing back to `tabindex="0"` with javascript.
+
+> Implementing Interactive Labels require custom javascript.
+
+<div class="sheet-example">
+	<span class="label label-dismissible label-secondary" tabindex="0">
+		<span class="label-item label-item-before">
+			<span class="sticker">
+				<span class="sticker-overlay">
+					<img alt="thumbnail" class="sticker-img" src="/images/thumbnail_dock.jpg">
+				</span>
+			</span>
+		</span>
+		<span class="label-item label-item-before">
+			<button class="btn btn-unstyled" type="button" tabindex="-1">
+				<svg class="lexicon-icon lexicon-icon-times-circle" focusable="false" role="presentation">
+					<use xlink:href="/images/icons/icons.svg#times-circle"></use>
+				</svg>
+			</button>
+		</span>
+		<span class="label-item label-item-before">
+			<a href="#1" role="button" tabindex="-1">
+				<svg class="lexicon-icon lexicon-icon-add-cell" focusable="false" role="presentation">
+					<use xlink:href="/images/icons/icons.svg#add-cell"></use>
+				</svg>
+			</a>
+		</span>
+		<span class="label-item label-item-expand">
+			<a href="#1" tabindex="-1">Interactive Label</a>
+		</span>
+		<span class="label-item label-item-after">
+			<button aria-label="Close" class="close" tabindex="-1" type="button">
+				<svg class="lexicon-icon lexicon-icon-times" focusable="false" role="presentation">
+					<use xlink:href="/images/icons/icons.svg#times"></use>
+				</svg>
+			</button>
+		</span>
+	</span>
+</div>
+
+```html
+<span class="label label-dismissible label-secondary" tabindex="0">
+	<span class="label-item label-item-before">
+		<span class="sticker">
+			...
+		</span>
+	</span>
+	<span class="label-item label-item-before">
+		<button class="btn btn-unstyled" type="button" tabindex="-1">
+			...
+		</button>
+	</span>
+	<span class="label-item label-item-before">
+		<a href="#1" role="button" tabindex="-1">
+			...
+		</a>
+	</span>
+	<span class="label-item label-item-expand">
+		<a href="#1" tabindex="-1">Interactive Label</a>
+	</span>
+	<span class="label-item label-item-after">
+		<button aria-label="Close" class="close" tabindex="-1" type="button">
+			...
+		</button>
+	</span>
+</span>
+```
+
+#### Interactive Label Examples
```

Comment:
hey @diegonvs I think we can simplify here and just add the example of Interactive Label without using MultiSelect.

Reply:
Done!